### DISCUSSION TITLE
Ultra 4K blueray of Flash Gordon (2020) will not rip.

### DISCUSSION THREAD

**Discussion by HaroldRobi** [automatic-ripping-machine/automatic-ripping-machine#824] [General]
Docker version of ARM reports the following when trying to rip Flash Gordon 4K BlueRay

MSG:2004,16777216,4,"Error 'Scsi error - ILLEGAL REQUEST:COPY PROTECTION KEY EXCHANGE FAILURE - KEY NOT PRESENT' occurred while issuing SCSI command A30..0020..03F to device 'SG:dev_21:3'","Error '%1' occurred while issuing SCSI command %4 to device '%2:%3'","Scsi error - ILLEGAL REQUEST:COPY PROTECTION KEY EXCHANGE FAILURE - KEY NOT PRESENT","SG","dev_21:3","A30..0020..03F"

Reproduce:
1,  Insert Blueray
2,  ARM begins reading disk.
3.  arm reports the above error repeatidly then ejects the disk
4. Verified Docker has current version of MakeMKV is v1.17.3
5. moved same blueray drive to Widnows machine
6. disk ripped successfully.

**Comment by HaroldRobi** [comment]
[FLASH_GORDON_UC_UHD.log](https://github.com/automatic-ripping-machine/automatic-ripping-machine/files/11228563/FLASH_GORDON_UC_UHD.log)


This is a copy of the flash Gordon log file.

**Comment by 1337-server** [comment]
I believe this is a makemkv issue and not an arm issue. It looks like that disc can't be decrypted yet.
